samples/net/cloud/google_iot_mqtt/src/main.c

```diff
@@ -19,6 +15,11 @@ LOG_MODULE_REGISTER(net_google_iot_mqtt, LOG_LEVEL_INF);
 #include <net/net_config.h>
 #include <net/net_event.h>
 
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(net_google_iot_mqtt, LOG_LEVEL_INF);
```

Comment:
Do we need to set LOG_LEVEL at all, at least in other networking samples that setting is omitted?

Reply:
It’s nor required anymore but if the Kconfig is not set for whatever reason it will prevent it from failing the build